pages/order.tsx

```diff
@@ -524,6 +524,7 @@ export default function OrderPage(props: OrderPageQuery & { error: any }) {
 
                       <FormattedMessage defaultMessage="Payment Fees not Considered" />
                       <FormattedMessage
+                        id="contribution.createdAt"
```

Comment:
@kewitz In languages with feminine/masculine forms, the translation depends on the context (what was created - an expense, a contrib, etc).